File: c#/uri/1005.cs
using System;
class URI{
    static void Main(string[] args){
        float a = float.Parse(Console.ReadLine().Trim());
        float b = float.Parse(Console.ReadLine().Trim());
        double c = ((a * 3.5) + (b * 7.5))/11;
        Console.WriteLine("MEDIA = {0:f5}", c);
    }
}

File: js/URI/1005.js
var input = require('fs').readFileSync('URI\\in', 'utf-8');
var lines = input.split('\n');
var a = parseFloat(lines.shift());
var b = parseFloat(lines.shift());
var c = (((a * 3.5) + (b * 7.5)) / 11);
var d = c.toFixed(5)
console.log(`MEDIA = ${d}`);